### PR TITLE
Add PLATOON SEmantic Data MOdels Of ENergy

### PR DESCRIPTION
**Pull request by sarra-ben-abbes**
Dear w3id team, 

We did a pull request to add the PLATOON ontologies in w3id. Could you please accept this project and make a redirection to this github: https://github.com/PLATOONProject/SEDMOON ? 

Thanks, 
Best regards

### CODE REVIEW

File: platoon/README.txt [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+There is only one namespace for PLATOON ontologies: https://w3id.org/platoon/ --> ;
+PLATOON Ontologies were designed to be consistent with W3C endorsed best practices on publishing data on the Web
+
+Source:
+
+https://w3id.org/platoon --> https://github.com/PLATOONProject/SEDMOON
+
+Contacts:
+
+Sarra BEN ABBES: benabbessarra@gmail.com Lynda TEMAL: lynda.temal@gmail.com Oumy SEYE: oumy.seye@gmail.com
```
Comment (TallTed) [inline platoon/README.txt:16]:
```suggestion
* Sarra BEN ABBES: benabbessarra@gmail.com 
* Lynda TEMAL: lynda.temal@gmail.com 
* Oumy SEYE: oumy.seye@gmail.com
```

**Comment by dgarijo** [comment]
@sarra-ben-abbes,
you are adding the ontologies diectly here. w3id does not host content. Please host the ontologies somewhere and then add just the redirects

**Comment by sarra-ben-abbes** [comment]
> @sarra-ben-abbes, you are adding the ontologies diectly here. w3id does not host content. Please host the ontologies somewhere and then add just the redirects

So, I delete the ontologies from here and put the redirection to the githut of platoon ontologies ?

**Comment by dgarijo** [comment]
Yes. Look at https://github.com/perma-id/w3id.org/tree/master/example for an example

**Comment by davidlehn** [comment]
> So, I delete the ontologies from here and put the redirection to the githut of platoon ontologies ?

When you remove them, please ensure they are not still in the git history (use rebase or other techniques).  Thanks.

**Comment by sarra-ben-abbes** [comment]
Dear w3id team,

I did another commit with only the readme, is it OK for you ? Thanks

**Comment by dgarijo** [comment]
@sarra-ben-abbes, no, you are still proposing to merge 22 files.

**Comment by sarra-ben-abbes** [comment]
Capture d’écran 2023-06-22 à 11 00 28

**Comment by sarra-ben-abbes** [comment]
> @sarra-ben-abbes, no, you are still proposing to merge 22 files.



> @sarra-ben-abbes, no, you are still proposing to merge 22 files.

I have this when I commit...

**Comment by davidlehn** [comment]
@sarra-ben-abbes You need to remove all the extra files, and fixup the commits so those files are not in the git history.  You can see all the extra files here:
- https://github.com/perma-id/w3id.org/pull/3412/commits
- https://github.com/perma-id/w3id.org/pull/3412/files

If you are familiar with git, it would involve removing files, committing the result and using 'git rebase' to squash or remove those extra commits.  If you want an easy path, you might try opening a new fresh PR that only adds .htaccess and README.md.  You also don't need both README.txt and README.md.  I'd suggest only having the .md.

**Comment by davidlehn** [comment]
I'm going to close this due to lack of activity.  Respond or reopen if needed.  If you still want this, please update to include commits with only .htaccess and README.md.  You can open a new PR too.

**Pull request closed without merging**